profiles/testbot.lua

```diff
@@ -19,6 +19,14 @@ function setup()
       use_turn_restrictions         = true
     },
 
+    classes = {[0] = "motorway", [1] = "toll"},
```

Comment:
is there any benefit to use keys here and below? `{ "motorway", "toll"}` should be ok